src/lib/swapper/swappers/ThorchainSwapper/utils/addSlippageToMemo.ts

```diff
@@ -6,14 +6,14 @@ import { MEMO_PART_DELIMITER } from './constants'
 import { assertIsValidMemo } from './makeSwapMemo/assertIsValidMemo'
 
 export const addSlippageToMemo = ({
-  expectedAmountOutThorBaseUnit,
+  emitAssetAmountThorBaseUnit,
```

Comment:
Matching the terminology here, since that was wrong from the start - this should be the emit asset amount, i.e expected_amount_out - fees.outbound

Reply:
Note this isn't just a name change - this is a name change *and* passing the correct amount around